src/stories/Configure.mdx

```diff
@@ -0,0 +1,364 @@
+import { Meta } from "@storybook/blocks";
```

Comment:
Please remove stories folder, in our project we will stories files inside the components folder. For example:

./components/Button

Button.tsx
Button.module.scss
Button.stories.ts

Reply:
Also please do not push mess into repository you can delete everything that is inside this folder, we won't need it